styleguide/checkstyle-suppressions.xml

```diff
@@ -12,4 +12,8 @@ suppressions PUBLIC "-//Puppy Crawl//DTD Suppressions 1.1//EN"
     checks="(EmptyLineSeparator|LineLength|MissingJavadocMethod|ParameterName)" />
   <suppress files=".*math[/\\]proto.*"
     checks="(CustomImportOrder|EmptyLineSeparator|LineLength|JavadocParagraph|MissingJavadocMethod|OverloadMethodsDeclarationOrder|SummaryJavadoc|UnnecessaryParentheses|OperatorWrap|JavadocMethod|JavadocTagContinuationIndentation)" />
+  <suppress files="wpilibj[/\\]src[/\\]generated.*"
+    checks="(OverloadMethodsDeclarationOrder|MethodName|UnusedImports)" />
```

Comment:
Why is it necessary to suppress unused imports?

Reply:
For controllers like StadiaController that pull in the HAL and usage reporting, but don't currently report anything because they don't have a number.